src/app/pages/dashboard/components/widget-network/widget-network.component.ts

```diff
@@ -484,15 +481,31 @@ export class WidgetNetworkComponent extends WidgetComponent implements OnInit, A
     return `${this.translate.instant('IP Address')}: ${this.getIpAddress(nic)}`;
   }
 
-  private getSpeedLabel(value: number): string {
-    const converted = filesize(Math.round(Math.abs(value)), { bits: true }).replace('bit', 'b');
-    return `${converted}/s`;
-  }
-
-  private splitValue(value: number): number {
-    if (value < 1024) {
-      return Number(value.toString().slice(0, 4));
+  protected getSpeedLabel(value: number): string {
+    let bits = Math.abs(value);
```

Comment:
Can we not get filesize pipe to work? I'd rather we not build our own code for this.